mungegithub/mungers/assign-unassign-handler.go

```diff
@@ -87,9 +88,18 @@ func (h AssignUnassignHandler) Munge(obj *github.MungeObject) {
 
 	toAssign, toUnassign := h.getAssigneesAndUnassignees(obj, comments, fileList, potentialOwners)
 	for _, username := range toAssign.List() {
-		obj.AssignPR(username)
+		err := obj.AssignPR(username)
+		if err != nil {
+			glog.Errorf("Could not assign %v to PR %v because %v", username, obj.Issue.Number, err)
+		}
+	}
+	for _, username := range toUnassign.List() {
+		err := obj.UnassignPR(username)
+		if err != nil {
+			glog.Errorf("Could not unassign %v from PR %v because %v", username, obj.Issue.Number, err)
+		}
 	}
-	obj.UnassignPR(toUnassign.List()...)
```

Comment:
see line 97.  Instead of removing them all at once, it's done a person by person basis to 1) be consistent with the assign call and 2) so that the error is specific to a single person that wrote `/unassign` rather than for a group